optuna/pruners/threshold.py

```diff
@@ -0,0 +1,116 @@
+import math
+from typing import Optional
+
+from optuna.pruners import BasePruner
+from optuna.pruners.percentile import _is_first_in_interval_step
+
+import optuna
```

Comment:
Please reorder the import lines based on [this rule](https://github.com/optuna/optuna/issues/514)?

Reply:
Sorry for violating the rule. :bow:
1ba33f6 I run `isort` to fix the import order.

Reply:
No. No problem 👍 Thank you for your swift fix.